python-proxy-on-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,263 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "newStorageAccountName": {
+            "type": "string",
+            "defaultValue": "uniquestorageaccountName",
```

Comment:
Can you remove the default value for storage account since these need to be unique DNS names?